src/main/java/com/eva/model/applicant/Applicant.java

```diff
@@ -0,0 +1,89 @@
+package com.eva.model.applicant;
+
+import java.util.Optional;
+import java.util.Set;
+
+import com.eva.model.person.Address;
+import com.eva.model.person.Email;
+import com.eva.model.person.Name;
+import com.eva.model.person.Person;
+import com.eva.model.person.Phone;
+import com.eva.model.tag.Tag;
+
+/**
+ * Represents the Applicant to be stored by Eva.
+ */
+public class Applicant extends Person {
+    protected Optional<InterviewDate> interviewDate;
+    protected ApplicationStatus applicationStatus;
```

Comment:
Minor comment, but maybe it could be private instead of protected? Since we would just use getter methods to access these fields.